Update draft-ietf-teas-5g-ns-ip-mpls.md (#203)
boucadair/5g-slice-realization: draft-ietf-teas-5g-ns-ip-mpls.md

   reservations.  By "fixed", we mean a value that stays constant over
   time, unless the 5G NSO communicates a change in slice bandwidth
   requirements, due to the creation or modification of a slice.  Note

>    that the "reservations" can be in the mind of the transport

Reviewer: ```suggestion
   that the "reservations" may be maintained by the transport
```
